drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -151,6 +148,7 @@ def items_without_answers():
             for item in items:
                 del item['feedback']
                 del item['zone']
+                item['displayName'] = self.ugettext(item['displayName'])
```

Comment:
@mattdrayer `displayName` may contain HTML markup (cf. [tests](https://github.com/open-craft/xblock-drag-and-drop-v2/blob/master/tests/integration/data/test_html_data.json#L24)) which would end up in the translation files (and would need to be recreated in individual translations). The same goes for zone titles and initial and final feedback. I'm not sure if this is a problem; just pointing it out for now in case it's relevant.